doc/user/zebra.rst

```diff
@@ -103,6 +103,14 @@ Besides the common invocation options (:ref:`common-invocation-options`), the
    the upper level daemons that can install v6 routes with v4
    nexthops.
 
+.. option:: --kernel-ext-learn
+
+   Signal to zebra that its operating in kernel external learn mode.
```

Comment:
I expect much broader documentation in this case, because as an operator, it's not clear when we should use it.